bin/_mocha

```diff
@@ -180,7 +180,7 @@ program
     `specify user-interface (${interfaceNames.join('|')})`,
     'bdd'
   )
-  .option('-w, --watch', 'watch files for changes')
+  .option('-w, --watch', 'watch files in the CWD for changes')
```

Comment:
please expand CWD to "current working directory" in both cases

Reply:
@boneskull good idea, done